### ISSUE TITLE
Create second database within the program and fast switch between databases

### ISSUE THREAD

**Issue by zelpi** [stashapp/stash#2723]
**Is your feature request related to a problem? Please describe.**
Create a second database within the program to separate collections. It seems possible by copying manually the program directory in explorer but the more elegant solution would be as part of the stash menu.

**Describe the solution you'd like**
Create a second (and maybe more) database(s) in stash and switch between the different databases with a mouse click.

**Describe alternatives you've considered**
Copy manually the stash directory into a second one and adjust the config-file. Didn‘t work for me…

**Comment by VonTittyslappen** [comment]
I agree totally, it would benefit those that want to keep separate databases of stuff, a database for the standard porn, another for Hentai (if you're of that persuasion) another for onlyfans stuff, another for cam model content as well.

The option should be there eventually for those that want to be able to switch databases easily.

**Comment by DogmaDragon** [comment]
You can already run multiple Stash instances with their own config files. Not only that allows you to have different configurations but also keep the generated content separate.
